bin/create-config.js

```diff
@@ -15,6 +15,15 @@ const pkg = JSON.parse(await fs.readFile(new URL("../package.json", import.meta.
 
 info(`${pkg.name}: v${pkg.version}\n`);
 
+process.on("uncaughtException", error => {
+    if (error instanceof Error && error.code === "ERR_USE_AFTER_CLOSE") {
+        // eslint-disable-next-line n/no-process-exit -- exit gracefully on Ctrl+C
+        process.exit(1);
```

Comment:
```bash
PS D:\repos\eslint-org\create-config\example> node ..\bin\create-config.js
@eslint/create-config: v1.10.0

√ What do you want to lint? · javascript
× How would you like to use ESLint? ·
```
Exiting directly doesn't seem like a good user experience. At the very least, we could output something like "Operation canceled"?